src/App.tsx

```diff
@@ -1,25 +1,32 @@
-import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+// App.tsx
+import {
+  BrowserRouter as Router, Route, Routes, Navigate,
+} from 'react-router-dom';
 import Sidebar from './components/sidebar/Sidebar';
 import SignUp from './pages/SignUp';
-import LogIn from './pages/LogIn';
+import Login from './pages/LogIn';
 import Projects from './pages/Projects';
 import Editor from './pages/editor/Editor';
-import './App.css';
+import Profile from './pages/Profile';
 
-const App = (): JSX.Element => (
-  <Router>
-    <div style={{ display: 'flex' }}>
-      <Sidebar />
-      <div style={{ flex: 1, padding: '16px', height: '100vh' }}>
-        <Routes>
-          <Route path="/signup" element={<SignUp />} />
-          <Route path="/login" element={<LogIn />} />
-          <Route path="/projects" element={<Projects />} />
-          <Route path="/editor" element={<Editor />} />
-        </Routes>
+function App(): JSX.Element {
+  return (
+    <Router>
+      <div style={{ display: 'flex' }}>
```

Comment:
```suggestion
      <div style={{ display: 'flex', height: '100vh' }}>
```

Otherwise the editor height is not set constant